packages/features/ee/impersonation/lib/ImpersonationProvider.ts

```diff
@@ -213,7 +214,10 @@ async function isReturningToSelf({ session, creds }: { session: Session | null;
 
   if (returningUser) {
     // Skip for none org users
-    if (returningUser.role !== "ADMIN" && !returningUser.organizationId) return;
+    const inOrg =
```

Comment:
This doesn’t make sense. 

The variable is inOrg but the first thing it checks is the user has no organizationId set. Is this correct?